frontend/src/components/auth/login/login.js

```diff
@@ -1,8 +1,25 @@
 import React, { useReducer, useEffect } from "react";
 import "./login.css";
 
-//state type
```

Comment:
@urvashi-code1255 replace the lines from [here](https://github.com/HITK-TECH-Community/Community-Website/pull/188/files#diff-d5c81cb0d6ea549df508bdd0e919acef604720ffcee342acea1c1170ed52f506L4) to [here](https://github.com/HITK-TECH-Community/Community-Website/pull/188/files#diff-d5c81cb0d6ea549df508bdd0e919acef604720ffcee342acea1c1170ed52f506R20) with 

```
// Show/Hide Functionality.
window.onload = function () {
  const togglePassword = document.querySelector('#togglePassword');
  const password = document.querySelector('#password');
  if (togglePassword) {
    togglePassword.addEventListener('click', function (e) {
      // toggle the type attribute
      if (password) {
        const type = password.getAttribute('type') == 'password' ? 'text' : 'password';
        password.setAttribute('type', type);
        // toggle the eye slash icon
        this.classList.toggle('fa-eye-slash');
      }
    });
  }
};
```

Reply:
Oh I have to make the variables also after loading the window. Thank you so much @Kajol-Kumari. Actually I have searched a lot but this thing hasn't come to my mind. Thank you :)